checkov/sast/record.py

```diff
@@ -14,6 +15,7 @@ def __init__(self,
                  check_name: str,
                  check_result: _CheckResult,
                  code_block: List[Tuple[int, str]],
+                 data_flow: List[Tuple[int, str]],
```

Comment:
what if its not taint mode? why always add this as an empty list? let's default it to None and not add it to the final json in that case